src/btreemap.rs

```diff
@@ -16,9 +16,9 @@
 //! Root node address           ↕ 8 bytes
 //! ----------------------------------------
 //! Length (number of elements) ↕ 8 bytes
-//! ----------------------------------------
+//! ---------------------------------------- <- Address 28 (PACKED_HEADER_SIZE)
```

Comment:
IIUC the changes in this file are unrelated to the MemoryManager changes right? Maybe worth updating the description accordingly or splitting this out into its own PR?

Reply:
Split into #47